### PR TITLE
Rating section components

### PR DESCRIPTION
**Pull request by jacinthechong**

### CODE REVIEW

File: client/src/review-components/Ratings.jsx [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
 function Ratings(props) {
-  return (
-  <div className='ratings'>
-    <h1>3.5 <Stars rating='3.5'/></h1>
 
+  let testData = {
+    fiveStar: 10,
+    fourStar: 6,
+    threeStar: 4,
+    twoStar: 1,
```
Comment (rlhutong) [inline client/src/review-components/Ratings.jsx:12]: Not sure why twoStar is assigned 1 while oneStar is assigned 2

**Review by rlhutong — CHANGES_REQUESTED** [review]
Would you confirm the comment in ratings.jsx before approval?

File: client/src/review-components/Ratings.jsx (hunk 1 above)
Comment (jacinthechong) [reply client/src/review-components/Ratings.jsx:12]: This is the test data for number of reviews, so there's 2 one-star reviews, 1 two-star review and so on. I haven't connected the server with API data yet

Comment (rlhutong) [reply client/src/review-components/Ratings.jsx:12]: Got it!

**Review by rlhutong — APPROVED** [review]

**Pull request merged**